content/engine/release-notes/25.0.md

```diff
@@ -48,6 +48,21 @@ For a full list of pull requests and changes in this release, refer to the relev
 
 ### New
 
+- The daemon now uses systemd's default `LimitNOFILE`. In earlier versions of
+  Docker Engine, this limit was set to `infinity`. This would cause issues with
+  recent versions of systemd, where the hard limit was increased, causing
+  programs that adjusted their behaviors based on ulimits to consume a high
+  amount of memory.
+
+  The new setting makes containers behave the same way as software running on
+  the host, but may cause some software that makes incorrect assumptions based
+  on the soft-limit to misbehave. To get the previous behavior, you can set
+  `LimitNOFILE=1048576`.
+
+  This change only affects build containers created with `docker build` when
+  using BuildKit with the `docker` driver.
+  [moby/moby#45534](https://github.com/moby/moby/pull/45534)
+
```

Comment:
Wording looks good!

Perhaps "too much" information, but we _could_ consider adding a note that future versions of containerd will also use the new default, after which all containers will use systemd's defaults.

For users that run systemd 240 and above and that run into the problem with `RLIMIT_NOFILE=infinity` in the systemd unit for containerd, we could mention that they can use a system `drop-in` or `override` file to customise the settings for their setup  (Flatcar has a great page on [describing this in more depth](https://www.flatcar.org/docs/latest/setup/systemd/drop-in-units/))

Reply:
OK, added these bits